src/Ensconce.Console/Arguments.cs

```diff
@@ -199,7 +210,27 @@ private static OptionSet GetOptionSet()
                     "dictionarySavePath=",
                     @"Specify a file to save the tag directory to (as JSON)",
                     s => DictionarySavePath = s
-                }
+                },
+                {
+                    "b|backup",
+                    @"Specify the source of the backup.  Required for the backup option",
```

Comment:
Help text is same as below - though option could be implicit if someone has specified source/destination

Reply:
your right, removing option